Resources/Prototypes/_DV/Entities/Objects/Fun/instruments.yml

```
@@ -56,3 +56,29 @@
       types:
         Blunt: 7
         Shock: 2
+
+- type: entity
+  parent: BaseKeyedInstrument
+  id: BloodRedSuperSynth
+  name: blood red super synthesizer
+  description: Covoered in red, fitting for a track calming track in some explosive missions.
+  components:
+  - type: ActivatableUI #Delta-V - Allows Super Synths to be played while being worn on the neck.
+    inHandsOnly: false
+    requireActiveHand: false
+  - type: Instrument
+    allowPercussion: true
+    allowProgramChange: true
+    respectMidiLimits: true #Delta-V - Enables limits to prevent people from crashing the server with black midis.
+  - type: Sprite
+    sprite: _DV/Objects/Fun/Instruments/redsupersynth.rsi
+    state: bloodredsynth
+  - type: Clothing # Delta-V - The straps on the super synth have a use now. Makes the blood red super synth wearable on the neck slot.
+    quickEquip: false
+    slots:
+    - neck
+    sprite: _DV/Objects/Fun/Instruments/redsupersynth.rsi
+  - type: Item
+    heldPrefix: bloodredsynth
+    size: Normal # Delta-V - Ensures that the sprite properly renders, had to add this because it wouldn't render properly on the ground and in UI/bag UI otherwise.
+    sprite: _DV/Objects/Fun/Instruments/redsupersynth.rsi
```

I suggest just to use `SuperSynthesizerInstrument` as the parent and override the sprite. That way, if the super synth changes for whatever reason, we won't need to touch this item as well. And we can get rid of all the DV comments. Also a typo (Covoered in red)

```suggestion
- type: entity
  parent: SuperSynthesizerInstrument
  id: BloodRedSuperSynth
  name: blood red super synthesizer
  description: Covered in red, fitting for a calming track in some explosive missions.
  components:
  - type: Sprite
    sprite: _DV/Objects/Fun/Instruments/redsupersynth.rsi
    state: bloodredsynth
  - type: Clothing
    quickEquip: false
    slots:
    - neck
    sprite: _DV/Objects/Fun/Instruments/redsupersynth.rsi
  - type: Item
    heldPrefix: bloodredsynth
    size: Normal
    sprite: _DV/Objects/Fun/Instruments/redsupersynth.rsi
```

---

Change is done :)